omniscient_fake/src/faker.py

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env python
+
+import rospy
+
+from geometry_msgs.msg import Point, Vector3, Pose, PoseArray
+from gazebo_msgs.msg import ModelStates
+from std_msgs.msg import Float64MultiArray, MultiArrayDimension, MultiArrayLayout
+from aquadrone_msgs.msg import SubState
+
+
+
+
+
+class Faker:
+    def __init__(self):
+        self.relative_pos_pub = rospy.Publisher("omni_faker", Float64MultiArray, queue_size = 1)
```

Comment:
The topic this posts to will be the same one the real vision node eventually uses, so it should have a neutral name:
vision_data?